cmd/virt-handler/BUILD.bazel

```diff
@@ -137,16 +137,6 @@ pkg_tar(
     package_dir = "/etc",
 )
 
-pkg_tar(
```

Comment:
Removing the nft files in a separate commit may make the review of this PR a little bit easier.
It would also better document the change in git history